pandas/src/algos_groupby_helper.pxi.in

```diff
@@ -669,6 +669,72 @@ def group_cumsum(numeric[:, :] out,
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
+def group_cummin(numeric[:, :] out,
+                 numeric[:, :] values,
+                 int64_t[:] labels,
+                 numeric[:, :] accum):
+    """
+    Only transforms on axis=0
+    """
+    cdef:
+        Py_ssize_t i, j, N, K, size
+        numeric val, min_val
+        int64_t lab
+
+    N, K = (<object> values).shape
+    accum = np.empty_like(accum)
+    accum[:] = _int64_max
```

Comment:
Thanks for the head-up @chris-b1! Would that mean creating 3 different `group_cummin`/`group_cummax` functions to accommodate  `int32`, `float32` and `float64`? It doesn't seem possible to get the `dtype` directly from the values array.